nautobot/dcim/tests/test_api.py

```diff
@@ -362,16 +362,15 @@ def setUpTestData(cls):
         cls.lt4 = LocationType.objects.get(name="Room")
 
         status_active = Status.objects.get(slug="active")
-        cls.site = Site.objects.first()
         tenant = Tenant.objects.create(name="Test Tenant")
 
-        loc1 = Location.objects.create(name="RTP", location_type=cls.lt1, status=status_active, site=cls.site)
-        loc2 = Location.objects.create(name="RTP4E", location_type=cls.lt2, status=status_active, parent=loc1)
-        loc3 = Location.objects.create(name="RTP4E-3", location_type=cls.lt3, status=status_active, parent=loc2)
-        loc4 = Location.objects.create(
-            name="RTP4E-3-0101", location_type=cls.lt4, status=status_active, parent=loc3, tenant=tenant
+        cls.loc1 = Location.objects.create(name="RTP", location_type=cls.lt1, status=status_active)
+        cls.loc2 = Location.objects.create(name="RTP4E", location_type=cls.lt2, status=status_active, parent=cls.loc1)
+        cls.loc3 = Location.objects.create(name="RTP4E-3", location_type=cls.lt3, status=status_active, parent=cls.loc2)
+        cls.loc4 = Location.objects.create(
+            name="RTP4E-3-0101", location_type=cls.lt4, status=status_active, parent=cls.loc3, tenant=tenant
         )
-        for loc in [loc1, loc2, loc3, loc4]:
+        for loc in [cls.loc1, cls.loc2, cls.loc3, cls.loc4]:
             loc.validated_save()
 
         # FIXME(jathan): The writable serializer for `Device.status` takes the
```

Comment:
Gonna have some fun merge conflicts to resolve now that #3225 is in `next`... sorry!